### PR TITLE
Modified the file for PR Request

### PR DESCRIPTION
**Pull request by sihab9874**
Please review the modification.

### CODE REVIEW

**Comment by binbhutto** [comment]
Please add another module.

**Comment by sihab9874** [comment]
> Please add another module.

working on it.

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Zzzz
```
Comment (binbhutto) [inline file1.txt]: Please capitalize all the letter.

**Review by binbhutto — APPROVED** [review]
Looks good.

File: file.txt [hunk 2]
```diff
@@ -3,4 +3,5 @@ bbb
 AAA
 BBB
 CCC
-DDD
\ No newline at end of file
+DDD
+Ppppp
```
Comment (binbhutto) [inline file.txt:7]: Looks okay.

**Review by binbhutto — APPROVED** [review]

**Pull request merged**